Hazel/src/Hazel/ImGui/ImGuiConsole.cpp

```diff
@@ -0,0 +1,249 @@
+#include "hzpch.h"
+#include "ImGuiConsole.h"
+
+#include "imgui.h"
+#include "examples/imgui_impl_glfw.h"
+#include "examples/imgui_impl_opengl3.h"
+
+namespace Hazel {
+
+	float ImGuiConsole::s_DisplayScale = 1.0f;
```

Comment:
Alot of these static members seem like they should be actual members of 'ImGuiConsole' and 'ImGuiConsole' should be the spdlog sink